README.md

```diff
@@ -69,31 +69,31 @@ of [Conda](https://docs.conda.io/en/latest/):
 1. [Installation and Usage](#installation-and-usage)
 1. [Configuration](#configuration)
 1. [Supported Rules](#supported-rules)
-   1. [Pyflakes (F)](#pyflakes)
-   1. [pycodestyle (E, W)](#pycodestyle)
-   1. [mccabe (C90)](#mccabe)
-   1. [isort (I)](#isort)
-   1. [pydocstyle (D)](#pydocstyle)
-   1. [pyupgrade (UP)](#pyupgrade)
-   1. [pep8-naming (N)](#pep8-naming)
-   1. [flake8-2020 (YTT)](#flake8-2020)
-   1. [flake8-annotations (ANN)](#flake8-annotations)
-   1. [flake8-bandit (S)](#flake8-bandit)
-   1. [flake8-blind-except (BLE)](#flake8-blind-except)
-   1. [flake8-boolean-trap (FBT)](#flake8-boolean-trap)
-   1. [flake8-bugbear (B)](#flake8-bugbear)
-   1. [flake8-builtins (A)](#flake8-builtins)
-   1. [flake8-comprehensions (C4)](#flake8-comprehensions)
-   1. [flake8-debugger (T10)](#flake8-debugger)
-   1. [flake8-import-conventions](#flake8-import-conventions)
-   1. [flake8-print (T20)](#flake8-print)
-   1. [flake8-quotes (Q)](#flake8-quotes)
-   1. [flake8-return (RET)](#flake8-return)
-   1. [flake8-tidy-imports (I25)](#flake8-tidy-imports)
-   1. [eradicate (ERA)](#eradicate)
-   1. [pygrep-hooks (PGH)](#pygrep-hooks)
-   1. [Pylint (PLC, PLE, PLR, PLW)](#pylint)
-   1. [Ruff-specific rules (RUF)](#ruff-specific-rules)
+   1. [Pyflakes (F)](#pyflakes-f)
+   1. [pycodestyle (E, W)](#pycodestyle-e-w)
+   1. [mccabe (C90)](#mccabe-c90)
+   1. [isort (I)](#isort-i)
+   1. [pydocstyle (D)](#pydocstyle-d)
+   1. [pyupgrade (UP)](#pyupgrade-up)
+   1. [pep8-naming (N)](#pep8-naming-n)
+   1. [flake8-2020 (YTT)](#flake8-2020-ytt)
+   1. [flake8-annotations (ANN)](#flake8-annotations-ann)
+   1. [flake8-bandit (S)](#flake8-bandit-s)
+   1. [flake8-blind-except (BLE)](#flake8-blind-except-ble)
+   1. [flake8-boolean-trap (FBT)](#flake8-boolean-trap-fbt)
+   1. [flake8-bugbear (B)](#flake8-bugbear-b)
+   1. [flake8-builtins (A)](#flake8-builtins-a)
+   1. [flake8-comprehensions (C4)](#flake8-comprehensions-c4)
+   1. [flake8-debugger (T10)](#flake8-debugger-t10)
+   1. [flake8-import-conventions (ICN)](#flake8-import-conventions-icn)
+   1. [flake8-print (T20)](#flake8-print-t20)
+   1. [flake8-quotes (Q)](#flake8-quotes-q)
+   1. [flake8-return (RET)](#flake8-return-ret)
+   1. [flake8-tidy-imports (I25)](#flake8-tidy-imports-i25)
+   1. [eradicate (ERA)](#eradicate-era)
+   1. [pygrep-hooks (PGH)](#pygrep-hooks-pgh)
+   1. [Pylint (PLC, PLE, PLR, PLW)](#pylint-plc-ple-plr-plw)
+   1. [Ruff-specific rules (RUF)](#ruff-specific-rules-ruf)
```

Comment:
We definitely need to auto-generate this too, it keeps getting stale :)

(Not asking you to do that, or for it to be part of this PR, just an observation!)

Reply:
No doubt!!

I can do that in this PR or split, your call. Rust is kind of fun!!

Reply:
Awesome! Separate PR would be best if you're up for it. (It can probably be part of `generate-rules-table` -- you can follow the logic there for how we find-and-replace sections within the README using these `BEGIN_PRAGMA` and `END_PRAGMA` constructs.)

Reply:
TOC handled in #1121